### PR TITLE
Implemented Level Popup

### PR DESCRIPTION
**Pull request by he-larious**
See issue #128

### CODE REVIEW

File: Scripts/Singletons/SceneSwapper.gd
Comment (m-lin1) [inline Scripts/Singletons/SceneSwapper.gd:149]: line 149 comment, wouldn't index = -1 only if old_scene can't be found? If it was the last level then it would be equal to scene_array.size(). The code checks out, but this comment might be confusing later on

**Review by m-lin1 — COMMENTED** [review]
Small change with line 149 comment.
Bug when exiting level via the level complete popup & going to Start Menu: outline of the level's Grid is visible.

File: Scripts/Singletons/SceneSwapper.gd
Comment (he-larious) [reply Scripts/Singletons/SceneSwapper.gd:149]: Comment out of date my bad

**Review by m-lin1 — APPROVED** [review]
YAAAAAAHOOOOOOO

**Comment by m-lin1** [comment]
waiting on another review before merging

**Comment by he-larious** [comment]
The PEP.draw() bug fix should be fine. I played around with it for a bit and it didn't break anything else

**Pull request merged**
